infotv/frontend/package.json

```diff
@@ -4,49 +4,46 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "build": "webpack --verbose --progress",
-        "watch": "webpack --verbose --progress --watch",
-        "release": "webpack -p",
-        "lint": "eslint src webpack.config.js --ext .js,.jsx",
+        "build": "webpack --progress",
+        "watch": "webpack --mode=development --progress --watch",
+        "release": "webpack --mode=production",
+        "lint:ts": "tslint src/**.ts*",
         "prettify": "prettier --write '**/*.js*'"
     },
     "author": "Aarni Koskela <akx@iki.fi>",
     "license": "MIT",
     "private": true,
     "devDependencies": {
-        "babel-eslint": "^8.2.3",
-        "eslint": "^4.19.1",
-        "eslint-config-airbnb": "^16.1.0",
-        "eslint-import-resolver-webpack": "^0.10.0",
-        "eslint-plugin-import": "^2.12.0",
-        "eslint-plugin-jsx-a11y": "^6.0.3",
-        "eslint-plugin-react": "^7.9.1",
-        "prettier": "^1.13.4"
+        "awesome-typescript-loader": "^5.0.0",
```

Comment:
Is there a reason to use `awesome-typescript-loader` instead of just `ts-loader`?